stories/topics/proc-azure-monitor-deployment-engine.adoc

```diff
@@ -0,0 +1,21 @@
+[id="azure-monitor-deployment-engine_{context}"]
+
+= Monitoring deployments on the {PlatformNameShort} Deployment Engine
+
+[role="_abstract"]
+
+The deployment engine displays information about your {AAPonAzureNameShort} deployment.
+You can monitor the progress of the deployment and you can cancel the deployment.
+
+When you begin deploying {AAPonAzureNameShort}, the Azure interface displays the *Overview* page for the deployment.
+
+. Click *Parameters and Outputs* in the *Settings* menu.
++
+Approximately 10 minutes into the deployment process, the *Outputs* section of the *Parameters and Outputs* page displays a link to the *`deploymentEngineUrl`*.
```

Comment:
The "Outputs" section shows nothing. If we want to find the Deployment Engine URL, we need to go to the "Parameters and Outputs" section of the deployed managed application after the Azure web shows "Your deployment is complete".

Reply:
I think she is referring to the outputs portion of the Parameters and Outputs page, where it says 4 outputs in this screenshot.  So it seems accurate to me.
![image](https://user-images.githubusercontent.com/49794041/225697487-16dfd1f5-b660-4947-997c-0c296b95c0e3.png)

Reply:
When I read the document, it linked me to the "Output" section on this screen. 
![image](https://user-images.githubusercontent.com/93540182/225942967-c4a8d1d9-6045-4307-adf1-4b492ed685b0.png)
 
On this screen, the "Output" was always empty even though Azure Web showed "Your deployment is complete!". @bhavenst , If we want to find the Deployment Engine URL, we need to go to the "Parameters and Outputs" section of the deployed managed application after the Azure web shows "Your deployment is complete". This is the exact screen as what you shared above.